ara/ui/templates/partials/host_index_row.html

```diff
@@ -1,51 +1,52 @@
+      {{ host.failed |add:host.unreachable }}
+    </a>
+    <a class="badge badge-pill badge-info ara-result-status-badge" title="Search for tasks with status: skipped" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=skipped#results"{% endif %}>
+      {{ host.skipped }}
+    </a>
+  </td>
   <td class="text-nowrap">{{ host.updated | format_date }}</td>
   <td class="text-nowrap">{{ host.playbook.duration | format_duration }}</td>
   <td>
     <a {% if not static_generation %}href="{% url 'ui:host_index' %}?name={{ host.name }}&latest=false" title="Search results for {{ host.name }}"{% endif %}>
       {{ host.name }}
     </a>
   </td>
-  <td>
-    {% if host.ok %}
-    <a class="badge badge-pill badge-success ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=ok#results" title="Search ok results for {{ host.name }}"{% endif %}>
-      {{ host.ok }} OK
-    </a>
-    {% endif %}
-    {% if host.changed %}
-    <a class="badge badge-pill badge-warning ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?changed=True#results" title="Search changed results for {{ host.name }}"{% endif %}>
-      {{ host.changed }} CHANGED
-    </a>
-    {% endif %}
-    {% if host.failed %}
-    <a class="badge badge-pill badge-danger ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=failed#results" title="Search failed results for {{ host.name }}"{% endif %}>
-      {{ host.failed }} FAILED
-    </a>
-    {% endif %}
-    {% if host.unreachable %}
-    <a class="badge badge-pill badge-danger ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=unreachable#results" title="Search unreachable results for {{ host.name }}"{% endif %}>
-      {{ host.unreachable }} UNREACHABLE
-    </a>
+  {% if host.playbook.name is not None %}
+  <td title="Search results for {{ host.playbook.name }}">
+    {% if not static_generation %}
+      <a href="{% url 'ui:host_index' %}?playbook_name={{ host.playbook.name }}">{{ host.playbook.name }}</a>
+    {% else %}
+      {{ host.playbook.name }}
     {% endif %}
-    {% if host.skipped %}
-    <a class="badge badge-pill badge-info ara-result-status-badge" {% if not static_generation %}href="{% url 'ui:host' host.id %}?status=skipped#results" title="Search skipped results for {{ host.name }}"{% endif %}>
-      {{ host.skipped }} SKIPPED
-    </a>
+  {% else %}
+  <td title="Search results for {{ host.playbook.path }}">
+    {% if not static_generation %}
+      <a href="{% url 'ui:host_index' %}?playbook_path={{ host.playbook.path }}">{{ host.playbook.path | truncatepath:50 }}</a>
+    {% else %}
+      {{ host.playbook.path | truncatepath:50 }}
     {% endif %}
+  {% endif %}
   </td>
```

Comment:
I was leveraging your work to do the same thing in the new tasks page (thanks again!) and was briefly confused due to being two ``<td>``'s but only a single ``</td>``. I understand why it isn't necessary but for the sake of clarity I'll duplicate the </td> inside the if/else block.